src/test/compile-fail/issue-19244-1.rs

```diff
@@ -12,5 +12,5 @@ const TUP: (usize,) = (42,);
 
 fn main() {
     let a: [isize; TUP.1];
-    //~^ ERROR attempted out-of-bounds tuple index
+    //~^ ERROR no field `1` on type `(usize,)`
```

Comment:
This error is clearly less good from my point of view. Before it was very clear but now, seems more confusing.